nixnet/_enums.py

```diff
@@ -2029,3 +2029,26 @@ class FrameType(enum.Enum):
     SPECIAL_DELAY = _cconsts.NX_FRAME_TYPE_SPECIAL_DELAY
     SPECIAL_LOG_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_LOG_TRIGGER
     SPECIAL_START_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_START_TRIGGER
+
+class Terminals(enum.Enum):
```

Comment:
Since the constants are prefixed with `TERM`, should this enum be called `Term`?

iirc that was how I decided the name for all of the other enums

Reply:
"Term" is ambiguous since we have both "Termination" and "Terminal" in the mix. For readability improvements mentioned below I suggest dropping the NX and spelling out terminal i.e TERMINAL_FRONTPANEL0.

Reply:
Also, iirc we don't tend to name the enums as a plural, so if it is expanded, it'd be `Terminal`.